GraphsV2/utils/DataParser/standardParser.js

```diff
@@ -0,0 +1,16 @@
+import { DEFAULT_BARGRAPH_ORIENTATION } from '../../../constants';
+
+export default ({ data, key, xColumn, yColumn, orientation }) => {
+    const map = [];
+    data.map((item) => {
+        map.push({
+            ...item,
+            'xVal': item[xColumn],
+            'yVal': item[yColumn]
+        });
+    });
+    return {
+        'uniqueKeys': DEFAULT_BARGRAPH_ORIENTATION === orientation ? ['yVal'] : ['xVal'],
```

Comment:
again we are using here customization where it should be generic? Please have a default of which values to set the unique keys by instead of orientation.